content/ansible/roles/requirements.yml

```diff
@@ -1,3 +1,5 @@
 roles:
 - name: manageiq.manageiq_vmdb
+  version: v1.0.0
 - name: manageiq.manageiq_automate
+  version: v1.0.0
```

Comment:
FYI, since we haven't changed this in so long, I'm fine with staying with more explicit versions.  We can loosen if it we find we need to do more releases.